flink-runtime/src/main/java/org/apache/flink/runtime/io/network/partition/BoundedBlockingSubpartition.java

```diff
@@ -268,9 +268,7 @@ public int unsynchronizedGetNumberOfQueuedBuffers() {
 
     @Override
     public int getNumberOfQueuedBuffers() {
-        synchronized (lock) {
-            return getBuffersInBacklogUnsafe();
-        }
```

Comment:
Shouldn't this change affect some unit test? Does it mean it doesn't have a test coverage?